src/envo/envo-edit.owl

```diff
@@ -5751,7 +5766,7 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_00000070> "\"constructed\" should probably be made something like a quality and this class obsoleted or filled only by inference")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_00000070> "constructed feature")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000070> "human construction"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_00000070> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01001813> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000996>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000070> <http://purl.obolibrary.org/obo/ENVO_01001813>)
```

Comment:
ah, the joy of opaque identifiers. Can't evaluate this statement without resolving the URI...